webcompat/static/js/lib/bugform.js

```diff
@@ -459,14 +466,50 @@ function BugForm() {
         this.stepsToReproduceField.val(function(idx, value) {
           return value.replace(/\[!\[[^\]]+\]\([^\)]+\)\]\([^\)]+\)$/, "");
         });
+
+        this.hasImage = false;
       }, this)
     );
   };
+
+  this.maybeUploadImage = _.bind(function(event) {
+    if (!this.hasImage) {
+      // nothing to do if there's no image, so form submission
+      // can happen regularly.
+      return;
+    }
+
+    event.preventDefault();
+
+    this.uploadImage(
+      // grab the data URI from background-image property, since it's already
+      // in the DOM: 'url("data:image/....")'
+      this.previewEl.get(0).style.backgroundImage.slice(5, -2),
+      _.bind(function(response) {
```

Comment:
Ummm.... I... I don't actually need this extra `_.bind`. Will test and remove -- thanks!